doc/dataview.md

```diff
@@ -0,0 +1,245 @@
+# DataView
+
```

Comment:
In general  as a rule in all the documentation please use the  Napi namespace e.g `Napi::DataView`.